app/src/main/res/values/strings.xml

```diff
@@ -182,7 +182,9 @@
     <string name="txtTakePicture">Take picture</string>
     <string name="txtSendAll">Upload all</string>
     <string name="txtNoData">No data</string>
-    <string name="txtPictureNeeded">Please take a picture first</string>
```

Comment:
Can you change this to something along the lines of:
> Wait... 

Please use &#8230; for the ellipsis

Reply:
@Karljoones  should I assign  "Wait..." to `txtPictureNeeded`. Is it what you meant ?

Reply:
Yes that is correct

Reply:
@Karljoones then all changes are made. I think this PR needs to be approved.